blog/authors.yml

```diff
@@ -4,4 +4,12 @@ chatdisabled:
   url: https://github.com/ChatDisabled
   image_url: https://github.com/ChatDisabled.png
   socials:
-    github: ChatDisabled
\ No newline at end of file
+    github: ChatDisabled
+  
+scorpion:
+  name: Scorpion
+  title: Volunteer for the Qbox Project
+  url: https://github.com/Scorpion7162
+  image_url: https://cdn.discordapp.com/avatars/625338242425815050/3a7b6dbbbde9851e2d739b0198170a05.webp?size=1024&format=webp&width=410&height=410
```

Comment:
You'll need to change the `image_url` as discord makes images auto expire after 24hrs (or less cant remember) and the image will no longer work. For example see how chat did his